src/firefly/js/templates/lightcurve/LcConverterFactory.js

```diff
@@ -19,36 +22,77 @@ import {makeLsstSdssPlotRequest} from './lsst_sdss/LsstSdssPlotRequests.js'; //L
  * @param {Object} params - mission specific parameters
  */
 
+/**
```

Comment:
The documentation below is very useful.  I am so glad you put the effort into adding it.